dev/ci/platform-windows.bat

```diff
@@ -48,7 +48,10 @@ call coq_platform_make_windows.bat ^
   -extent=i ^
   -parallel=p ^
   -jobs=2 ^
-  -switch=d || GOTO ErrorCopyLogFilesAndExit
+  -switch=d ^
+  -override-dev-pkg="coq=%CI_PROJECT_URL%/-/archive/%CI_COMMIT_SHA%/coq-%CI_COMMIT_SHA%.tar.gz" ^
+  -override-dev-pkg="coqide=%CI_PROJECT_URL%/-/archive/%CI_COMMIT_SHA%/coqide-%CI_COMMIT_SHA%.tar.gz" ^

```

Comment:
shouldn't that be
```suggestion
  -override-dev-pkg="coqide=%CI_PROJECT_URL%/-/archive/%CI_COMMIT_SHA%/coq-%CI_COMMIT_SHA%.tar.gz" ^
```
?

Reply:
ok, but I'll wait for CI to run before pushing a new commit